source/docs/articles/sites/files/temp-files.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Temporary File Management with Multiple Application Containers
+description: Learn how to work with temporary files in distributed environments.
+category:
+- getting-started
+keywords: tmp, temp files, tmp files, temporary files, multiple application containers, distributed environments
+---
+Live sites on Professional plans and above have multiple [application containers](/docs/articles/sites/all-about-application-containers).  Pantheon's distributed system means that requests are spread between all of the available application servers, which is part of how we help the site scale. However, the `/tmp` directory on one instance is not able to access the `/tmp` resources on another application server. There are several solutions available that allow you to upload files and create temporary directories on Pantheon.
+
+## Plupload
+
+ [Plupload](http://www.plupload.com/) is a multi-runtime file uploader and has been used successfully with the Pantheon platform. To do so, change the default upload directory from `tmp` to `private://tmp` in  `filefield_sources_plupload.module`. You may also need to add this line to run through `files/private/tmp` every few hours and delete old files to keep it from piling up:   
```

Comment:
@ari-gold Think I should link to this one? https://www.drupal.org/project/plupload